inc/Engine/CriticalPath/CriticalCSS.php

```diff
@@ -551,4 +551,27 @@ public function is_async_css_mobile() {
 
 		return (bool) $this->options->get( 'async_css_mobile', 0 );
 	}
+
+	/**
+	 * Get list of CSS files to be excluded from async CSS.
+	 *
+	 * @since 3.6.1
+	 *
+	 * @return array An array of URLs for the CSS files to be excluded.
+	 */
+	public function get_exclude_async_css() {
+		/**
+		 * Filter list of async CSS files
+		 *
+		 * @since 2.10
+		 * @author Remy Perona
+		 *
+		 * @param array $exclude_async_css An array of URLs for the CSS files to be excluded.
+		 */
+		$exclude_async_css = (array) apply_filters( 'rocket_exclude_async_css', [] );
+		$exclude_async_css = array_filter( $exclude_async_css );
+		$exclude_async_css = array_flip( array_flip( $exclude_async_css ) );
+
+		return $exclude_async_css;
```

Comment:
Let's return here instead of assigning and then returning.

```suggestion
		return array_flip( array_flip( $exclude_async_css ) );
```